commands-yml/commands/session/geolocation/set-geolocation.yml

```diff
@@ -1,6 +1,6 @@
 ---
 name: Set Geolocation
-short_description: Set the current geo location
+short_description: Set the current geo location. (altitude works on only Android real devices)
```

Comment:
What about mentioning the point in https://github.com/appium/appium/pull/12906/files#diff-bd878dad3adb5356b1a3c1aae26d6d85R82 rather than short description?

Reply:
I understand your points, but there is spec between clients and the server in my understanding, so users might miss its info there.

I'd mean it's same to here. (but above info is more details..)
https://github.com/appium/appium/blob/4785026e65cd5b96929895b7d1cd5f1e2cff0b32/commands-yml/commands/device/network/gsm-signal.yml#L3

@mykola-mokhnach 
Can I have your comments?

Reply:
`altitude works on real Android devices only` is fine with me

Reply:
I meant this case addressed only a parameter like https://github.com/appium/appium/blob/920197b7ba6b4fa1beda0c2f86bcc921667b2f98/commands-yml/commands/device/recording-screen/start-recording-screen.yml#L116 and https://github.com/appium/appium/blob/c596608bfde4ed0dc763952712e940000cfb2e67/commands-yml/commands/device/interactions/lock.yml#L78 rather than this command is available only for an android emulator like your link.

I'm fine we'll address parameters before describing parameters if it helps client users.
You can add `description` section to mention such details, too.

---

> spec between clients and the server

For Appium client/server developers, I would recommend referring https://github.com/appium/appium-base-driver/blob/master/lib/protocol/routes.js , btw.

Reply:
@KazuCocoa @mykola-mokhnach 
Thanks for your comments.

@KazuCocoa 
Thanks for your examples, I agree with you and will follow that way.